provider/maas/Makefile

```diff
@@ -8,10 +8,7 @@ bridgescript.go: add-juju-bridge.py Makefile
 	$(RM) $@
 	echo -n '// This file is auto generated. Edits will be lost.\n\n' >> $@
 	echo -n 'package maas\n\n' >> $@
-	echo -n '//go:generate make -q\n\n' >> $@
```

Comment:
It was never used. We don't have a top-down rule that runs `go generate`. If I change the script I have (for ever) just run `make` in the provider/maas directory, then commit those changes.